File: targets/python-web/vulns/vuln33_csv.py
from flask import Blueprint, request, Response
bp = Blueprint("vuln33", __name__)

@bp.get("/vuln33/export")
def export():
    note = request.args.get("note", "")
    # VULN:VULN-33:CWE-1236:taint user data written into CSV without neutralising formulas
    return Response("id,note\n1," + note + "\n", mimetype="text/csv")

File: targets/jsts-web/src/vulns/vuln33_csv.ts
import { Router } from "express";
export const router = Router();
router.get("/vuln33/export", (req, res) => {
  const note = String(req.query.note || "");
  // VULN:VULN-33:CWE-1236:taint user data written into CSV without neutralising formulas
  res.type("csv").send("id,note\n1," + note + "\n");
});
